redshift/resource_redshift_user_test.go

```diff
@@ -356,6 +359,11 @@ resource "redshift_user" "with_email" {
   password = "Foobarbaz1"
 }
 
+resource "redshift_user" "with_hashed_password" {
+  name = "hashed_password"
+  password = "md5153c434b4b77c89e6b94f12c5393af5b"
```

Comment:
I think this is the wrong hash?  Changed the pw to match the other user.

https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_USER.html

`select 'md5' || md5('Foobarbaz2' || 'hashed_password');`

```suggestion
  password = "md5ad3b897bab2474bc7e408326cb18c42f"
```

Reply:
@hoxu I used a different password deliberately (ending in 2 instead of 1), so that possible bugs in other provider machinery couldn't leak into this and cause a false positive.  Excessive paranoia never hurt any software engineers :D